commit 9989e81679580012054811d9106345fcde68c11f
Author: Jonas Devlieghere <jonas@devlieghere.com>
Date:   Thu Jun 18 14:04:34 2020 -0700

    [Sphinx] Adjust for source_parsers deprecation in Sphinx 3.0
    
    Update the Sphinx configuration for the removal of source_parsers in
    Sphinx 3.0. The variable has been deprecated since version 1.8.
    
    > Version 1.8 deprecates and version 3.0 removes the source_parsers
    > configuration variable that was used by older recommonmark versions.
    
    https://www.sphinx-doc.org/en/master/usage/markdown.html
    
    Differential revision: https://reviews.llvm.org/D75284

diff --git b/llvm/docs/conf.py a/llvm/docs/conf.py
index 13bbc826de8..c92ede3ea44 100644
--- b/llvm/docs/conf.py
+++ a/llvm/docs/conf.py
@@ -26,16 +26,15 @@ from datetime import date
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.intersphinx', 'sphinx.ext.todo', 'recommonmark']
+extensions = ['sphinx.ext.intersphinx', 'sphinx.ext.todo']
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
 # The suffix of source filenames.
-source_suffix = {
-    '.rst': 'restructuredtext',
-    '.md': 'markdown',
-}
+source_suffix = ['.rst', '.md']
+
+source_parsers = {'.md': 'recommonmark.parser.CommonMarkParser'}
 
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
